qiime/metadata.py

```diff
@@ -14,12 +16,30 @@ def __init__(self, dataframe):
     def __repr__(self):
         return self._dataframe.to_json()
 
+    @staticmethod
+    def load(path):
+        try:
+            df = pd.read_csv(path, sep='\t', index_col=0, dtype=object)
```

Comment:
Thanks, @antgonza. The last point is particularly annoying with AG samples as most are numeric-like (e.g., 10317.000004210) but are not safe to treat as floats.